src/ripple/basics/BasicConfig.h

```diff
@@ -40,15 +39,18 @@ using IniFileSections = std::map<std::string, std::vector<std::string>>;
 /** Holds a collection of configuration values.
     A configuration file contains zero or more sections.
 */
-class Section : public beast::unit_test::detail::const_container<
-                    std::map<std::string, std::string, boost::beast::iless>>
+class Section
 {
 private:
     std::string name_;
+    std::map<std::string, std::string> lookup_;
     std::vector<std::string> lines_;
     std::vector<std::string> values_;
     bool had_trailing_comments_ = false;
 
+    using const_iterator =
+        typename std::map<std::string, std::string>::const_iterator;
```

Comment:
Nit: The `typename` keyword isn't needed for this declaration. Also consider `decltype(lookup_)` for concision and less redundancy.